src/tree/v2/application/GroupingItem.ts

```diff
@@ -34,7 +34,11 @@ export class GroupingItem implements ResourceGroupsItem {
         public readonly resources: ApplicationResource[]) {
     }
 
-    readonly id: string = this.label;
+    readonly id: string = `groupings/${this.label}`;
+
+    isAncestorOf(id: string): boolean {
+        return this.resources.some(resource => id === resource.id || id.startsWith(resource.id));
```

Comment:
`startsWith` will still return true for `id === resourceId`, so technically that first equality comparison isn't needed. Though, having it adds some clarity (i.e., I had to go look and see if `startsWith` actually did that or not). I'm fine with either.